File: Client/src/content/life_6_tests.py
def run_tests_next_cell_value() :
    fn = next_cell_value.__name__
    def test(c, expected):
        test_function(fn, expected, next_cell_value(testGrid, c), ["testGrid", c]);
    test(0, False); #Currently: False, 4
    test(19, False); #Currently: True,4
    test(30, True); #Currently: True,2
    test(44, True); #Currently: False,3
    test(59, False); #Currently: False,4
    test(60, True); #Currently: True,3
    test(399, False); #Currently: False,4
    all_tests_passed()

File: Client/src/content/life_6_tests.cs
static void RunTests()
{
    testNextCellValue(0, false);
    testNextCellValue(19, false);
    testNextCellValue(30, true);
    testNextCellValue(44, true);
    testNextCellValue(59, false);
    testNextCellValue(60, true);
    testNextCellValue(399, false);
    AllTestsPassed();
}

private static void testNextCellValue(int c, bool expected)
{
    TestFunction(nameof(NextCellValue), expected, NextCellValue(exampleCells, c), "exampleCells", c);
}

